kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -379,6 +381,19 @@ object KyuubiConf {
     .transform(_.toLowerCase(Locale.ROOT))
     .createWithDefault(SaslQOP.AUTH.toString)
 
+  val FRONTEND_REST_BIND_HOST: OptionalConfigEntry[String] = buildConf("frontend.rest.bind.host")
+    .doc("Hostname or IP of the machine on which to run the REST frontend service.")
+    .version("1.0.0")
```

Comment:
this version  may need to follow the version of Kyuubi

Reply:
Hi, yes, this PR is under draft state. It's not ready for review. Opening for verifying CI.